pkg/config/config.go

```diff
@@ -116,10 +114,6 @@ func KyvernoServiceAccountName() string {
 	return kyvernoServiceAccountName
 }
 
-func KyvernoDeploymentName() string {
```

Comment:
Why changing the deployment name only? I find these envs quite useful as they can be accessed globally.

Reply:
Accessing them globally is useful but if we forgot to set the env var we don't see it. Shall we panic if the env var is missing ?

Reply:
That's not a bad idea. I say we stick with the env and panic when it's not set.